packages/swap/src/routes/swap.ts

```diff
@@ -183,6 +183,7 @@ router.get(
         swapDepositChannel?.srcChainExpiryBlock?.toString(),
       estimatedDepositChannelExpiryTime:
         estimatedDepositChannelExpiryTime?.valueOf(),
+      isExpired: swapDepositChannel?.isExpired ?? false,
```

Comment:
i would say its fine to have this as `undefined` if there is no deposit channel for the swap
```suggestion
      isExpired: swapDepositChannel?.isExpired,
```